dotCMS/src/integration-test/java/com/dotcms/content/elasticsearch/business/ESContentFactoryImplTest.java

```diff
@@ -363,4 +393,92 @@ public void testDeleteVersion_KeepPermissions() throws DotSecurityException, Dot
         }
     }
 
+    @Test
+    @UseDataProvider("dateTimeTestCases")
+    public void testIndexSearchFilteringByDateTime(final TestCase testCase){
+
+        LocalDateTime today, yesterday, tomorrow;
+
+        final LocalDateTime now = LocalDateTime.now();
+
+        today = now;
+        yesterday = today.minusDays(1).withHour(0).withMinute(0).withSecond(0);
+        tomorrow = today.plusDays(1).withHour(23).withMinute(59).withSecond(59);
+
+        buildLuceneQueryAndTest(testCase, today, yesterday, tomorrow);
+    }
+
+    @Test
+    @UseDataProvider("timeTestCases")
+    public void testIndexSearchFilteringByTime(final TestCase testCase){
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [JUnit tests should include assert() or fail()](https://app.codacy.com/app/dotCMS/core/pullRequest?prid=4545506)